### PR TITLE
fix(core): `DropdownContext` close previous dropdown after new one is opened

### LINKED ISSUE #2524: 🐞 - `DropdownContext` close previous dropdown after new one opening
### Which @taiga-ui/* package(s) are the source of the bug?

core

### Is this issue blocking you?

Blocking

### Description

1. Open https://taiga-ui.dev/directives/dropdown-context#contextMenu
2. Make many right-click on the table

Screenshot 2022-08-30 at 15 38 14

### Angular version

any

### Taiga UI version

3.0.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2524

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo-integrations/cypress/tests/kit/dropdown-context/dropdown-context.spec.ts [hunk 1]
```diff
@@ -19,6 +19,19 @@ describe(`DropdownContext`, () => {
             .matchImageSnapshot(`01-opened-context-menu`, {capture: `viewport`});
     });
 
+    it(`closes previous dropdown after new one is opened`, () => {
+        cy.get(`#contextMenu`).find(`tr`).eq(1).rightclick(`left`);
+        cy.get(`tui-dropdown`).should(`have.length`, 1);
+        cy.get(`#contextMenu`).matchImageSnapshot(`02-1-opened-dropdown`);
+
+        cy.get(`#contextMenu`).find(`tr`).eq(2).rightclick(`center`);
+        cy.get(`tui-dropdown`).should(`have.length`, 1);
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/tests/kit/dropdown-context/dropdown-context.spec.ts:28]: It passes locally but fails in CI 🤔

Comment (nsbarsukov) [reply projects/demo-integrations/cypress/tests/kit/dropdown-context/dropdown-context.spec.ts:28]: not problem of this test

File: projects/demo/src/modules/components/island/island.template.html [hunk 2]
```diff
@@ -40,12 +40,12 @@
...
                     <li class="tui-list__item">
                         <code>.tui-island__paragraph_link</code>
-                        : parahraph with links. Margin top
+                        : paragraph with links. Margin top
                         <code>16px</code>
                     </li>
                     <li class="tui-list__item">
                         <code>.tui-island__paragraph_button</code>
-                        : parahraph with buttons. Margin to
+                        : paragraph with buttons. Margin to
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/island/island.template.html:48]: how did you find that typo :D

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/components/island/island.template.html (hunk 2 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/components/island/island.template.html:48]: I noticed it before (when I was refactoring `Action`). Then I forgot about it.
And now (when I need to trigger changes inside `demo` somehow) I've recalled about it ))

**Pull request merged**